### ISSUE TITLE
Conversion of ScrapeConfig to VMScrapeConfig with scheme field failed

### ISSUE THREAD

**Issue by barthy1** [VictoriaMetrics/operator#1148]
In case if `scheme` value is specified in Prometheus ScrapeConfig format (HTTP or HTTPS), I have error in VM operator during conversion ScrapeConfig into VMScrapeConfig - `VMScrapeConfig.operator.victoriametrics.com \"test\" is invalid: spec.scheme: Unsupported value: \"HTTPS\": supported values: \"http\", \"https\"` 
But HTTP/HTTPS are only valid scheme values for ScrapeConfig. I guess http/https would work for conversion, but it fails validation for ScrapeConfig format.

Sample ScrapeConfig
```
apiVersion: monitoring.coreos.com/v1alpha1
kind: ScrapeConfig
metadata:
  name: test
spec:
  scheme: HTTPS
  staticConfigs:
    - targets:
        - test.target.com
```

**Comment by f41gh7** [comment]
It looks weird:

https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go#L273
https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/types.go#L482

 For v1 scheme accepts http and https, for v1alpha it's HTTP and HTTPS. 

 We could add workaround for this case, but issue also must be addressed to the `prometheus-operator`.

**Comment by f41gh7** [comment]
Bug fix was included to the [v0.49.1](https://github.com/VictoriaMetrics/operator/releases/tag/v0.49.1) release
